contrail_api_cli_extra/clean/fip.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from sys import exit
+import argparse
+
+from contrail_api_cli.command import Option, Arg
+from contrail_api_cli.utils import printo
+from contrail_api_cli.resource import Collection, Resource
+from contrail_api_cli.exceptions import ResourceNotFound, CollectionNotFound
+
+from ..utils import server_type, CheckCommand
+from refs import CleanRefs
+
+class CleanBrokenFips(CheckCommand):
+    """
+    """
+    description = "Clean for broken floating IPs"
+
+    pool = Option(help="floating IP pool ref",
+                  nargs="?")
+    fip_refs = Arg(help="list of refs [fip0, fip1, fip2, fip3, ...]",
+                   nargs="*", default=[])
+    cassandra_servers = Option(help="cassandra server list' (default: %(default)s)",
+                               nargs='+',
+                               type=server_type,
+                               default=['localhost:9160'])
```

Comment:
Would be nice to add `contrail_api_cli_extra.utils.CassandraCommand` so that all commands that need cassandra can use it